projects/lua/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: "https://github.com/lua/lua"
+language: c
+primary_contact: "roberto@inf.puc-rio.br"
+auto_ccs:
+  - "fuzz@llua.org"
```

Comment:
@DavidKorczynski seems like a typo, shouldn't it be `lua.org`?

Reply:
you're absolutely right, just did a PR https://github.com/google/oss-fuzz/pull/5215